1-development/list_posts_by_rating/src/main/java/com/mixfar/posts/utils/enums/ErrorCode.java

```diff
@@ -0,0 +1,30 @@
+package com.mixfar.posts.utils.enums;
+
+public enum ErrorCode {
+    OK(200, "OK"),
+    CREATED(201, "CREATED"),
+    DELETED(201, "DELETED"),
```

Comment:
Why did you use this status code for ```DELETED```?

Reply:
The problem of `copy&paste` appears. I change the reason but not the code. For a `DELETED` status code, the code should be `200` for a successful delete or a `204` if we don´t return a content.